doc/classes/CompositorEffect.xml

```diff
@@ -35,21 +35,21 @@
 			var depth_buffer = render_scene_buffers.get_texture("render_buffers", "depth")
 			[/codeblock]
 		</member>
-		<member name="effect_callback_type" type="int" setter="set_effect_callback_type" getter="get_effect_callback_type" enum="CompositorEffect.EffectCallbackType">
+		<member name="effect_callback_type" type="int" setter="set_effect_callback_type" getter="get_effect_callback_type" enum="CompositorEffect.EffectCallbackType" default="4">
 			The type of effect that is implemented, determines at what stage of rendering the callback is called.
 		</member>
-		<member name="enabled" type="bool" setter="set_enabled" getter="get_enabled">
+		<member name="enabled" type="bool" setter="set_enabled" getter="get_enabled" default="true">
 			If [code]true[/code] this rendering effect is applied to any viewport it is added to.
 		</member>
-		<member name="needs_motion_vectors" type="bool" setter="set_needs_motion_vectors" getter="get_needs_motion_vectors">
+		<member name="needs_motion_vectors" type="bool" setter="set_needs_motion_vectors" getter="get_needs_motion_vectors" default="true">
```

Comment:
The changes to this file seem wrong. Since doctool didn't fail the pipeline, it seems to be a problem of how the fields are initialized (maybe your code isn't at fault but simply reveals the issue?).

Reply:
If a class has not been used doctool can't find the default values.